matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/session/room/EventRelationsAggregationTask.kt

```diff
@@ -374,4 +418,61 @@ internal class DefaultEventRelationsAggregationTask @Inject constructor(
             Timber.e("## Cannot find summary for key $reactionKey")
         }
     }
+
+    private fun handleVerification(realm: Realm, event: Event, roomId: String, isLocalEcho: Boolean, relatedEventId: String, userId: String) {
+        val eventSummary = EventAnnotationsSummaryEntity.where(realm, relatedEventId).findFirst()
```

Comment:
You should create a method `getOrCreate()` in the EventAnnotationsSummaryEntityQuery